src/puppetlabs/trapperkeeper/services/authorization/authorization_core.clj

```diff
@@ -142,9 +142,16 @@
         get-br-list (fn [match-str]
                       (map #(Integer/parseInt (second %))
                            (re-seq #"\$(\d+)" match-str)))
+        get-certname-list (fn [rule]
```

Comment:
I'm contemplating moving this to a proper outer function, mainly so I could write a couple of unit tests for it.